src/main/java/com/spikes2212/control/FeedForwardSettings.java

```diff
@@ -12,7 +12,13 @@ public class FeedForwardSettings {
     /**
      * Empty FeedForwardSettings, which effectively make the FeedForwardController do nothing.
      */
-    public static final FeedForwardSettings EMPTY_FFSETTINGS = new FeedForwardSettings(0, 0);
+    public static final FeedForwardSettings EMPTY_FFSETTINGS = new FeedForwardSettings(0, 0,
+            FeedForwardController.ControlMode.LINEAR_POSITION);
```

Comment:
add (or replace this with) an option to get an empty ffSettings of any control mode, to be used with `Namespace.addFeedForwardNamespace`.